tools/postinstall-android.js

```diff
@@ -66,29 +66,12 @@ const libProjectRoot = path.join('plugins', 'com.salesforce', 'src', 'android',
 const appProjectRoot = path.join('platforms', 'android');
 
 console.log('Moving Salesforce libraries to the correct location');
-shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceAnalytics'), appProjectRoot);
-shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceSDK'), appProjectRoot);
-shelljs.cp('-R', path.join(libProjectRoot, 'SmartStore'), appProjectRoot);
-shelljs.cp('-R', path.join(libProjectRoot, 'MobileSync'), appProjectRoot);
-shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceHybrid'), appProjectRoot);
-
-console.log('Fixing Gradle dependency paths in Salesforce libraries');
-const oldSalesforceAnalyticsDep = "api(project\(\":libs:SalesforceAnalytics\"\))";
-const oldSalesforceSdkDep = "api(project\(\":libs:SalesforceSDK\"\))";
-const oldSmartStoreDep = "api(project\(\":libs:SmartStore\"\))";
-const oldMobileSyncDep = "api(project\(\":libs:MobileSync\"\))";
-replaceTextInFile(path.join(appProjectRoot, 'SalesforceSDK', 'build.gradle.kts'), oldSalesforceAnalyticsDep, 'api(project\(\":SalesforceAnalytics\"\))');
-replaceTextInFile(path.join(appProjectRoot, 'SmartStore', 'build.gradle.kts'), oldSalesforceSdkDep, 'api(project\(\":SalesforceSDK\"\))');
-replaceTextInFile(path.join(appProjectRoot, 'MobileSync', 'build.gradle.kts'), oldSmartStoreDep, 'api(project\(\":SmartStore\"\))');
-replaceTextInFile(path.join(appProjectRoot, 'SalesforceHybrid', 'build.gradle.kts'), oldMobileSyncDep, 'api(project\(\":MobileSync\"\))');
+shelljs.cp('-R', path.join(libProjectRoot, 'mobile_sdk'), appProjectRoot);
```

Comment:
By referencing the entirety of the MSDK Gradle workspace, things get a lot simpler here.  Also, MSDK's build configuration will be exactly as it is when building it outside this composite workspace.  That should aid in preventing future surprises 😏